bucket/xmplay.json

```
@@ -0,0 +1,38 @@
+{
+    "version": "3.8.3.4",
+    "license": "freeware",
+    "homepage": "https://www.un4seen.com/xmplay.html",
+    "description": "a Win32 audio player. Initially, when XMPlay was first released in 1998, only the XM format was supported, hence the name \"XMPlay\". Now supporting formats as OGG, MP3, MP2, MP1, WAV, AIFF, CDA, and more with Winamp plugin and its own plugin support.",
```

No one need histrorical lesson

```suggestion
    "description": "Win32 audio player with support for Winamp plugins.",
```